front-end/src/store/transcriptUploadedSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const transcriptUploadedSlice = createSlice({
```

Comment:
Maybe rename to transcriptUpload. We can use this slice for both the boolean, and the eventual transcript when we havea  get request. Also personally, since this is essentially an object, I don't think putting it in past tense is very clear.